src/components/form/_form.scss

```diff
@@ -107,36 +106,36 @@
 
       &::before {
         background: $color-off-white;
-        border-radius: $border-radius / 2;
-        height: 18px;
+        border-radius: .125rem;
+        height: .25rem;
         left: 0;
         top: 50%;
         transform: translateY(-53%);
-        width: 18px;
+        width: .25rem;
```

Comment:
Are the `width` and the `height` sizes intrinsically linked? If so, I would consider a local variable in the file itself. If one changing should impact the other, they should be centrally scored in `_forms.scss` variable.